en/components/dialog.md

```diff
@@ -77,13 +77,25 @@ There are four Text properties for the Dialog, Title, Message, Left Button, and
 - {settingsLabel}
 - Important {labelText}
 
+### Dialog Naming
+
+Dialog components are hidden at runtime until code opens them. The code generator uses the name given to the dialog in Sketch, to assign a name to the dialog in the HTML. For example, if the dialog was named: Dialog (will be changed to lower case) in Sketch the below bold #dialog will be added to the tag.
+
+<igx-dialog **#dialog**
```

Comment:
This tag looks incomplete.  Is that intentional?  And shouldn't it be marked as code?